tests/drivers/sensor/ina230/src/ina230_emul.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2023 North River Systems Ltd
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Emulator for the TI INA230 I2C power monitor
+ */
+#pragma once
```

Comment:
More common is:

<code>
#ifndef HEADERNAME_H
#define HEADERNAME_H
</code>

Reply:
I thought I saw it more in the codebase, but checking it looks like there are only 8 other instances.  Have replaced with standard header guards in the latest push.